doc/dev_guide/building_from_source.rst

```diff
@@ -38,43 +38,43 @@ the preferences can differ. But strategically the steps are always the same.
      option in step 5. For all platforms, this is ``python`` version 2.7+
      (but not 3.x). You need the following Python modules:
 
-       * `pyyaml <https://pypi.python.org/pypi/PyYAML>`_ version 3.10
-       * `argparse <https://pypi.python.org/pypi/argparse>`_ version 1.1
-       * `msgpack-python <https://pypi.python.org/pypi/msgpack-python>`_ version 0.4.6
-       * `gevent <https://pypi.python.org/pypi/gevent>`_ version 1.1.2
-       * `six <https://pypi.python.org/pypi/six>`_ version 1.8.0
+     * `pyyaml <https://pypi.python.org/pypi/PyYAML>`_ version 3.10
+     * `argparse <https://pypi.python.org/pypi/argparse>`_ version 1.1
+     * `msgpack-python <https://pypi.python.org/pypi/msgpack-python>`_ version 0.4.6
+     * `gevent <https://pypi.python.org/pypi/gevent>`_ version 1.1.2
+     * `six <https://pypi.python.org/pypi/six>`_ version 1.8.0
 
    To install all required dependencies, follow the instructions for your OS:
 
    * For Debian/Ubuntu, say:
 
      .. code-block:: console
 
-        $ apt install -y build-essential cmake make coreutils sed \
+        $ apt install -y build-essential git cmake make coreutils sed \
               autoconf automake libtool zlib1g-dev \
               libreadline-dev libncurses5-dev libssl-dev \
               libunwind-dev libicu-dev \
               python python-pip python-setuptools python-dev \
-              python-msgpack python-yaml python-argparse python-six python-gevent
+              python-msgpack python-yaml python-six python-gevent
 
    * For RHEL/CentOS (versions under 8)/Fedora, say:
 
      .. code-block:: console
 
-         $ yum install -y gcc gcc-c++ cmake make coreutils sed \
+         $ yum install -y gcc gcc-c++ git cmake make coreutils sed \
                autoconf automake libtool zlib-devel \
                readline-devel ncurses-devel openssl-devel \
                libunwind-devel libicu-devel \
                python python-pip python-setuptools python-devel \
-               python-msgpack python-yaml python-argparse python-six python-gevent
+               python-msgpack python-yaml python-six python-gevent
```

Comment:
That's is not so trivial considering that it is Python 2 packages on CentOS 7, but Python 3 ones on CentOS 8 / Fedora. I would just suggest to use `pip --user -r test-run/requirements.txt` or `pip -r test-run/requirements.txt` from a virtualenv environments. We can mention that the same can be installed with packages, but, to be honest, if one want, (s)he will do it anyway based on the requirements.txt content. So, let's stick to `pip` for simplicity.